Sources/RayTracer.Library/Lights/DirectionalLight.cs

```diff
@@ -14,21 +14,22 @@ public class DirectionalLight : ILight, ISerializable<DirectionalLight>
 
     public DirectionalLight(Vector3 direction)
         : this(direction, ColorRGB.White)
-    { }
+    {
+    }
 
     public DirectionalLight(Vector3 direction, ColorRGB color)
     {
         Direction = direction;
         Color = color;
     }
 
-    public ColorRGB PaintPoint(IIntersectable shape, Vector3 point)
+    public ColorRGB PaintPoint(IIntersectable shape, Vector3 normal)
```

Comment:
we indeed take the `Vector3 point` in the interface instead of the normal, and in this case where we also need normal in `PaintPoint` it's better to switch the method signature to
`ColorRGB PaintPoint(IIntersectable shape, in IntersectionResult result)`